src/contexts/StateContextProvider.tsx

```diff
@@ -95,6 +97,8 @@ const STATE_DEFAULT: Readonly<StateContextType> = Object.freeze({
     startQuery: () => null,
 });
 
+const INITIAL_QUERY_PROGRESS = 0;
```

Comment:
How do you feel about renaming it as `QUERY_PROGRESS_VALUE_MIN` just so we're consistent with the naming in LogExportManager?